articles/how-to-prepare-for-coding-interviews/index.md

```diff
@@ -0,0 +1,45 @@
+
+Whether you are a student or a working professional, effective preparation for software engineer interviews is essential if you want a job at a good company. *This tutorial covers everything you need to know to prepare for coding interviews in exceptional tech companies*.
+
+### Table of contents
+
+1.  [The interview process](#the-interview-process)
+2.  [The first round](#the-first-round)
+3.  [Technical interview](#technical-interview)
+4.  [HR interview](#hr-interview)
+5.  [Conclusion](#conclusion)
+
+### The interview process
+Reputed tech companies have a rigorous interview process to select the right candidate for the job. The first round is usually an online coding test conducted on platforms such as [Hackerrank](https://www.hackerrank.com/), [HackerEarth](https://www.hackerearth.com/), [Mettl](https://mettl.com/), etc. It consists of coding questions or a combination of coding questions and [MCQs](https://www.indiabix.com/online-test/aptitude-test/)(Multiple Choice Questions). Once the candidate clears the first round, they schedule a [telephonic interview](https://www.monsterindia.com/career-advice/phone-interview-tips-10-keys-to-succeed-in-a-telephonic-interview-6485.html) or an [onsite interview](https://www.algrim.co/302-onsite-interview) (The candidate is requested to go to the company office and attend a series of interviews). These interviews can be classified as technical or HR interviews. In technical interviews, you are asked questions about your resume, work experience(If any), fundamentals of computer science, Algorithms and Data structures, etc. HR interviews test your behavior, your ability to work well in a team, and whether you will be able to thrive in the company.
```

Comment:
Mention a timeframe required for preparing for the interview. A person would want to know how long it would take to prepare for these interviews. Also, point them to a roadmap for preparing for such interviews.